src/oemof/solph/processing.py

```diff
@@ -589,7 +589,7 @@ def detect_scalars_and_sequences(com):
 
     def move_undetected_scalars(com):
         for ckey, value in list(com["sequences"].items()):
-            if isinstance(value, str):
+            if isinstance(value, (str, int, float, np.number)):
```

Comment:
```suggestion
            if isinstance(value, (str, numbers.Number)):
```
This would work for all numeric types, not just int, float, and numpy types.